pkg/oc/admin/diagnostics/diagnostics/cluster/network/in_pod/util/log.go

```diff
@@ -23,7 +23,7 @@ func (l *LogInterface) LogNode(kubeClient kclientset.Interface) {
 	l.LogSystem()
 	l.LogServices()
 
-	l.Run("brctl show", "bridges")
+	l.Run("ip link", "interfaces")
```

Comment:
Do we still need `bridge-utils` package in node image? If not, we can remove this package from `images/dind/node/Dockerfile` and `images/node/Dockerfile`

Reply:
hack/dind-cluster.sh uses brctl too, so that would need to be fixed

Reply:
Given that we don't use linux bridges for anything at all any more, I think we can just drop this line